GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -104,7 +146,45 @@ - (void)verifyAccountDetailsInteractivelyWithOptions:(GIDSignInInternalOptions *
                 format:@"Your app is missing support for the following URL schemes: %@",
      [unsupportedSchemes componentsJoinedByString:@", "]];
   }
-  // TODO(#397): Start the incremental authorization flow.
+  NSURL *redirectURL = [NSURL URLWithString:[NSString stringWithFormat:@"%@:%@",
+                                             [schemes clientIdentifierScheme],
+                                             kBrowserCallbackPath]];
+
+  NSMutableDictionary<NSString *, NSString *> *additionalParameters = [@{} mutableCopy];
+  additionalParameters[kIncludeGrantedScopesParameter] = @"true";
+  if (options.configuration.serverClientID) {
+    additionalParameters[kAudienceParameter] = options.configuration.serverClientID;
+  }
+  if (options.loginHint) {
+    additionalParameters[kLoginHintParameter] = options.loginHint;
+  }
+  if (options.configuration.hostedDomain) {
+    additionalParameters[kHostedDomainParameter] = options.configuration.hostedDomain;
+  }
+
+  [additionalParameters addEntriesFromDictionary:
+      [GIDEMMSupport parametersWithParameters:options.extraParams
+                                   emmSupport:kEMMVersion
+                       isPasscodeInfoRequired:NO]];
+  additionalParameters[kSDKVersionLoggingParameter] = GIDVersion();
+  additionalParameters[kEnvironmentLoggingParameter] = GIDEnvironment();
+
+  NSMutableArray *scopes;
+  for (GIDVerifiableAccountDetail *detail in options.accountDetailsToVerify) {
+    NSString *scopeString = [detail scope];
+    if (scopeString) {
+      [scopes addObject:scopeString];
+    }
+  }
+
+  // TODO(#405): Use request variable to present request and process response.
+  __unused OIDAuthorizationRequest *request =
```

Comment:
It may be worthwhile to follow up with our colleagues to see if there is anything we can do now. I.e., can we send these requests? Is there a sandbox, even if incomplete, where we can start the web flow?

Reply:
Follow upped and it seems that there is still nothing for now but should be ready towards end of May.